smartdevicelink/sdl_hmi
# Navigation Subscription Buttons Implementation

Implements/Fixes #182 

This PR is **Ready** for review.

### Testing Plan
- Connect and activate a navigation app.
- Subscribe to a new navigation button.
- Click on "Nav Buttons" button to open list of nav buttons.
- Click on a button and verify the button events are received by the mobile app.

### Summary
Creates a new menu like view to house all of the navigation subscription buttons. Long and short presses are supported. 

### CLA
